src/contrib/cluster/Akka.Cluster.Sharding/ClusterShardingGuardian.cs

```diff
@@ -297,7 +297,7 @@ public sealed class StartProxy : INoSerializationVerificationNeeded
             });
         }
 
-        private ReplicatorSettings GetReplicatorSettings(ClusterShardingSettings shardingSettings)
+        internal static ReplicatorSettings GetReplicatorSettings(ClusterShardingSettings shardingSettings)
```

Comment:
Made this `internal` in order to make it easy to test the _real_ `ReplicatorSettings` that we're passing into DData - straight from the source.